packages/openapi3/test/tsp-openapi3/generate-type.test.ts

```diff
@@ -63,16 +63,20 @@ const testScenarios: TestScenario[] = [
   { schema: { $ref: "#/Path/To/Some/Model" }, expected: "Model" },
   { schema: { $ref: "#/Path/To/Some/Model.Prop" }, expected: "Model.Prop" },
   // arrays
-  { schema: { type: "array", items: { type: "string" } }, expected: "Array<string>" },
+  { schema: { type: "array", items: { type: "string" } }, expected: "(string)[]" },
   {
     schema: { type: "array", items: { type: "array", items: { type: "string" } } },
-    expected: "Array<Array<string>>",
+    expected: "((string)[])[]",
```

Comment:
Are those test run pre prettier? I would have expected this to be `string[][]`

Reply:
Yes...I only run prettier on the final file currently 😞

Reply:
ok, i'd say maybe it is better we test the final output but however you feel

Reply:
I updated these tests to run the formatter on the wrapped results...I don't love it because technically if the formatting does something weird to both the actual/expected results I won't know it. But the formatter does get ran eventually anyway so I don't feel too strongly about it - it is definitely easier to read the test case and understand what the final result should be now.

I plan to eventually write more tests that would be ran on the output of `generateMain` rather than just `generateTypeFromSchema` - the latter did catch a couple issues though, so I think there is value for testing at the individual generator level.